README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
Debian 8.6, 8.7 correspond to Ubuntu 14.04.1, 14.04.2.
`Debian-8` or `Ubuntu-14.04` implies that the OS is up to date.
`Ubuntu-14` means `Ubuntu-14.04 and Ubuntu-14.10` which are the different releases.
`Ubuntu-14` may be misleading the users that the role is tested or does suuport Ubuntu-14.10.

BTW, Ubuntu 14.10 has an official package of openjdk8 and ppa https://launchpad.net/~openjdk-r/+archive/ubuntu/ppa is available for Ubuntu 14.04.